nibe/connection/nibegw.py

```diff
@@ -523,6 +550,24 @@ def _encode(self, obj, context, path):
     MODBUS40=0x20,
 )
 
+
+RmuWriteIndex = Enum(
+    Int8ub,
+    TEMPORARY_LUX=0x02,
+    OPERATIONAL_MODE=0x04,
+    FUNCTIONS=0x05,
+    TEMPERATURE=0x06,
+)
+
+RmuTemporaryLux = Enum(
```

Comment:
I'd better use existing mapping for this.
See coil 48132

Reply:
Right i can drop these enums since there is correspondance to the registers. We cant use them during parse, but they are easily looked up.